Jenkins pipeline cigroups parallel

brianseeders opened the pull request:
test

brianseeders committed:
Pipeline

brianseeders committed:
WIP some work for parallelization with ciGroups

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](http://jenkins.localhost/job/kibana-automation-pipeline-brianseeders/1/)

brianseeders commented:
retest

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](http://jenkins.localhost/job/kibana-automation-pipeline-brianseeders/2/)

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](http://jenkins.localhost/job/kibana-automation-pipeline-brianseeders/28/)

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](http://jenkins.localhost/job/kibana-automation-pipeline-brianseeders/30/)

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](http://jenkins.localhost/job/kibana-automation-pipeline-brianseeders/31/)

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](http://jenkins.localhost/job/kibana-automation-pipeline-brianseeders/33/)

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](http://jenkins.localhost/job/kibana-automation-pipeline-brianseeders/35/)

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](http://jenkins.localhost/job/kibana-automation-pipeline-brianseeders/36/)

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](http://jenkins.localhost/job/kibana-automation-pipeline-brianseeders/37/)

brianseeders committed:
Fix xpack kibana install dir, and add some debugging

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](http://jenkins.localhost/job/kibana-automation-pipeline-brianseeders/38/)

brianseeders committed:
Attempt to quick fix a few tests

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](http://jenkins.localhost/job/kibana-automation-pipeline-brianseeders/39/)

elasticmachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](http://jenkins.localhost/job/kibana-automation-pipeline-brianseeders/40/)

brianseeders committed:
Recombine test groups, and try runbld again

brianseeders committed:
Mostly cleanup, and fix failed_tests reporting to hopefully work for both pipeline and non-pipeline

brianseeders committed:
Fix typo in shell script

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](http://jenkins.localhost/job/kibana-automation-pipeline-brianseeders/41/)

brianseeders committed:
Remove some debug code

brianseeders committed:
Add support for changing es transport.port during testing via TEST_ES_TRANSPORT_PORT

brianseeders committed:
Fix test that uses hard-coded es transport port and add it back in to parallel groups

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](http://jenkins.localhost/job/kibana-automation-pipeline-brianseeders/42/)

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](http://jenkins.localhost/job/kibana-automation-pipeline-brianseeders/43/)

brianseeders committed:
Disable checks reporter again for now

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](http://jenkins.localhost/job/kibana-automation-pipeline-brianseeders/44/)

brianseeders committed:
Set env var for TEST_ES_TRANSPORT_PORT in pipeline

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](http://jenkins.localhost/job/kibana-automation-pipeline-brianseeders/45/)

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](http://jenkins.localhost/job/kibana-automation-pipeline-brianseeders/46/)

brianseeders committed:
Update Jenkinsfile for shorter testrunner labels

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/kibana-automation-pipeline-brianseeders/4/)

brianseeders committed:
Fix another hard-coded transport port

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/kibana-automation-pipeline-brianseeders/5/)

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](http://jenkins.localhost/job/kibana-automation-pipeline-brianseeders/49/)

elasticmachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](http://jenkins.localhost/job/kibana-automation-pipeline-brianseeders/50/)

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/kibana-automation-pipeline-brianseeders/7/)

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](http://jenkins.localhost/job/kibana-automation-pipeline-brianseeders/51/)

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/kibana-automation-pipeline-brianseeders/8/)

brianseeders committed:
Fix a new test with hard-coded URLs

brianseeders committed:
Jenkinsfile cleanup and fix one of the groups

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/kibana-automation-pipeline-brianseeders/9/)

brianseeders committed:
Fix double slash

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/kibana-automation-pipeline-brianseeders/10/)

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](http://jenkins.localhost/job/kibana-automation-pipeline-brianseeders/52/)

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](http://jenkins.localhost/job/kibana-automation-pipeline-brianseeders/53/)

elasticmachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](http://jenkins.localhost/job/kibana-automation-pipeline-brianseeders/54/)

brianseeders committed:
Testing vault credentials on jenkins server

brianseeders committed:
Add a non-existent credential

brianseeders committed:
Revert "Add a non-existent credential"

This reverts commit 0dc234c465a5483b1a994cb510a182fef766e9cc.

brianseeders committed:
Try github-checks-reporter again

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/kibana-automation-pipeline-brianseeders/14/)

brianseeders committed:
github-checks-reporter should only run for elastic/kibana, forks won't work

brianseeders committed:
Clean up some debug code

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/kibana-automation-pipeline-brianseeders/15/)

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](http://jenkins.localhost/job/kibana-automation-pipeline-brianseeders/55/)

brianseeders committed:
Changing names around to try to make BlueOcean UI a little better

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](http://jenkins.localhost/job/kibana-automation-pipeline-brianseeders/56/)

brianseeders committed:
Add more stages

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](http://jenkins.localhost/job/kibana-automation-pipeline-brianseeders/57/)

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/kibana-automation-pipeline-brianseeders/18/)

brianseeders committed:
Make some changes to stage structure to mirror a nested example from CloudBees

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](http://jenkins.localhost/job/kibana-automation-pipeline-brianseeders/58/)

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/kibana-automation-pipeline-brianseeders/19/)

brianseeders committed:
Handle TODOs, and some cleanup in Jenkinsfile

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](http://jenkins.localhost/job/kibana-automation-pipeline-brianseeders/59/)

elasticmachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/kibana-automation-pipeline-brianseeders/20/)

brianseeders committed:
Pass GIT_BRANCH when started without GHPRB, fix branch check

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/kibana-automation-pipeline-brianseeders/21/)

brianseeders committed:
Fix mailer problem and add code that ensures all tests are in cigroups back in

elasticmachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/kibana-automation-pipeline-brianseeders/22/)

brianseeders committed:
Test adding worker/job name to junit report paths

brianseeders committed:
Remove some duplication from ci_setup scripts

brianseeders committed:
Fix unit test that uses junit path

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/kibana-automation-pipeline-brianseeders/24/)

brianseeders committed:
Don't reinstall node every time setup_env is run

brianseeders committed:
Fix yarn install logic

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/kibana-automation-pipeline-brianseeders/25/)

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/kibana-automation-pipeline-brianseeders/26/)

brianseeders committed:
Fix another unit test that uses junit output dir

brianseeders committed:
Download latest ES snapshot after kibana builds

elasticmachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/kibana-automation-pipeline-brianseeders/28/)

brianseeders committed:
Make sure junit reports are always processed

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/kibana-automation-pipeline-brianseeders/29/)

brianseeders committed:
Add two failing tests for testing purposes

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/kibana-automation-pipeline-brianseeders/30/)

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](http://jenkins.localhost/job/kibana-automation-pipeline-brianseeders/61/)

brianseeders committed:
Add support to Jenkinsfile for kibana build e-mails

brianseeders committed:
Remove some debug code for email sending

brianseeders committed:
Change JOB env handling in junit paths and move it to a sub-directory

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/kibana-automation-pipeline-brianseeders/32/)

brianseeders committed:
Revert "Add two failing tests for testing purposes"

This reverts commit 5715203e26922a93483feb0ebb8bb3fdcc3daf8c.

brianseeders committed:
Fix junit report path in test

brianseeders committed:
Don't send kibana emails on build abort

brianseeders committed:
Revert "Revert "Revert "[ci] compress jobs for CI stability" (#44584)""

This reverts commit 078ac2897fbb2e32db08990e3cbbd8cb65b6f536.

brianseeders committed:
Address PR feedback, formatting and use built-in url formatting library

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/kibana-automation-pipeline-brianseeders/33/)

Commit:
disable another flaky suite (#45323) (#45330)

Commit:
fix(code/frontend): fix document search result from (#45236)

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/kibana-automation-pipeline-brianseeders/34/)

Commit:
fix(code/frontend): should show updating instead of cloning when updating (#45238)

darnautov committed:
[ML] Data Frames: Cards as links (#45254)

* [ML] Data Frames: Cards as links

* [ML] PR remarks

jloleysens committed:
[Console] SQL template with triple quote in completion (#45248)

* SQL template with triple quote in completion

* Slight update to SQL query template and bugfix for collapse/expand of request bodies

* Add comment and slight update to triple quote expansion

* Updated tests after changes to newline behaviour

* Restore old backslash triple quote expansion behaviour

weltenwort committed:
[Logs UI] Interpret finished analysis jobs as healthy (#45268)

This changes the job health check of the log analysis page to consider finished jobs as healthy. This situation is relevant when the job is non-continuous, i.e. created with a definite end date.

At the same time it ensures that datafeeds that failed to be started due to empty indices lead to the jobs being considered unhealthy.

fixes #45180

jgowdyelastic committed:
[ML] Fixing wizard validation delay (#45265)

* [ML] Fixing wizard validation delay

* changes based on review

jgowdyelastic committed:
[ML] Fixing empty time range when cloning jobs (#45286)

* [ML] Fixing empty time range when cloning jobs

* removing if statement

sulemanof committed:
[Region Map] Fix loading default vector map and base layer setting (#43858)

* Fix loading default vector layer

* Move layers loading to vis initialization

* Move layers loading to editor initialization

kertal committed:
Convert value provided to PhraseValueInput to string to catch Exception (#45259)

jloleysens committed:
[Console] Update indentation behaviour (#45249)

* SQL template with triple quote in completion

* Slight update to SQL query template and bugfix for collapse/expand of request bodies

* Add comment and slight update to triple quote expansion

* Updated tests after changes to newline behaviour

* Restore old backslash triple quote expansion behaviour

* Handle some more tricky cases, but make sure to preserve newlines.
Update tests for new cases too

* Make regex more specific
Expanded tests

* Now make it x-browser

* Slight refactor and renaming

* Update comment

peteharverson committed:
[ML] Fixes display of matching modules in index data visualizer (#45261)

* [ML] Fixes display of matching modules in index data visualizer

* [ML] Update following review on adding recognizer check

brianseeders committed:
Fix path formatting for functional test

brianseeders committed:
Add email sending back in to Jenkinsfile

dmlemeshko committed:
Functional tests: convert more test/services to TS (#45176)

* convert more test/services to TS

* Update test/functional/services/combo_box.ts

Co-Authored-By: Tre' <wayne.seymour@elastic.co>

* Update test/functional/services/combo_box.ts

Co-Authored-By: Tre' <wayne.seymour@elastic.co>

* Update test/functional/services/combo_box.ts

Co-Authored-By: Tre' <wayne.seymour@elastic.co>

* fix lint error

flash1293 committed:
[Graph] Type persistence (#44985)

XavierM committed:
[SIEM] Inject/apply KQL changed in refresh button (#45065)

* wip to apply kql when refresh on hosts page

* refactor to have less re-render

* add network and timeline page

* fix/add unit testing

* from review remove any effect from render

* clean up + review II

* review II + bug fixes

* review III

pheyos committed:
[ML] Add multi metric job wizard test (#45279)

This PR adds functional UI tests to create a machine learning job using the multi metric wizard.

thompsongl committed:
Upgrade EUI to 13.8.1 (#45052)

* eui to 13.8.0

* euibutton ts updates

* euicontextmenupanel ts updates

* misc ts updates

* src snapshot updates

* x-pack snapshot updates

* 13.8.1

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/kibana-automation-pipeline-brianseeders/36/)

brianseeders committed:
Fix another unit test with path problem

elasticmachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/kibana-automation-pipeline-brianseeders/37/)
